spec/unit/shoryuken/error_handler_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+require 'airbrake/shoryuken/error_handler'
+
+module Airbrake
+  ##
+  # Provides tests for the Shoryuken integration.
+  module Shoryuken
```

Comment:
Would you mind unwrapping this? We don't use this style in this project. I think `RSpec.describe Airbrake::Shoryuken::ErrorHandler` should suffice.